modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/Actions.scss

```diff
@@ -1,5 +1,15 @@
 @import 'atlas-variables';
 
+.icons-list ul li {
+	color: $gray-600;
+	cursor: pointer;
+
+	&:hover {
+		background-color: #f0f5ff;
+		color: #272833;
```

Comment:
Can we use standard color variables here? If not, it's not a big deal to leave this as it is.

Reply:
As far as I saw, these colors are widely used without a variable. I found #f0f5ff in a variables file, but not related to FDS. Any clue on this are welcome.

I would even say that we use variables not many times...chance to a big refactor?

Reply:
@ugeortiz Do we have standard colors for hover text and background? If we don't on DXP level, we can add some on DSM level.